customer-config/pnp/pnp.edge-synchronizer-config.v1.json

```diff
@@ -10,7 +10,11 @@
   "$defs": {
     "Entity": {
       "type": "object",
-      "required": [],
+      "required": [
+        "name",
+        "collectionName",
+        "dfoEntityType"
```

Comment:
I do not think we need these dfo entity types in Edge Synchronizer: 
- EXTERNAL_DEVICE_ITEM
- PRINT_COMMAND
- ENTITY_CHANGES
- ENTITY_VALIDATION_RESULT
- ASSORTMENT_POLICY